src/OpenTelemetry.Collector.Dependencies/DependenciesCollector.cs

```diff
@@ -35,28 +35,16 @@ public class DependenciesCollector : IDisposable
         /// </summary>
         /// <param name="options">Configuration options for dependencies collector.</param>
         /// <param name="tracer">Tracer to record traced with.</param>
-        /// <param name="sampler">Sampler to use to sample dependnecy calls.</param>
+        /// <param name="sampler">Sampler to use to sample dependency calls.</param>
         public DependenciesCollector(DependenciesCollectorOptions options, ITracer tracer, ISampler sampler)
```

Comment:
Why have both sampler and custom sampler?

Reply:
good point. It seems there is no reason to have custom sampler now. And if someone wants it, they can use `DependenciesCollector` constructor to provide it. Will remove one from options

Reply:
sampler is globally registered one. CustomSampler is for filtering of a calls to Zipkin or such. CustomSampler falls back to one in constructor when doen't know what to do with the request. Also custom sampler works on raw http object so can sample based on url. BUT it is specific to the implementation